content/spin/kv-store-tutorial.md

```diff
@@ -102,13 +86,13 @@ $ spin new http-go spin-key-value
 
 ## Configuration
```

Comment:
Could we make this title more specific so it acts as a better guide to what is to come?